src/internal/storage/chunk/transform.go

```diff
@@ -47,33 +50,38 @@ func Create(ctx context.Context, opts CreateOptions, ptext []byte, createFunc fu
 
 // Get calls getFunc to retrieve a chunk, then verifies, decrypts, and decompresses the data.
 // cb is called with the uncompressed plaintext
-func Get(ctx context.Context, client Client, cache kv.GetPut, ref *Ref, cb kv.ValueCallback) error {
-	if err := getFromCache(ctx, cache, ref, cb); err == nil {
-		return nil
-	}
+func Get(ctx context.Context, client Client, cache kv.GetPut, deduper *miscutil.WorkDeduper, ref *Ref, cb kv.ValueCallback) error {
```

Comment:
I thought we were going to move the cache out of here into the dataReader, so that this code just focused on the transformation, now that the caching is more complicated.

Reply:
I'll add a TODO for this. I think we should focus on what is needed for now.